Demo/Common/IntegrationExample.swift

```diff
@@ -64,6 +73,36 @@ internal final class IntegrationExample: APIClientAware {
             }
         }
     }
+    
+    private func setupSession() {
+        let request = SessionSetupRequest()
+        apiClient.perform(request) { [weak self] result in
+            guard let self = self else { return }
+            switch result {
+            case let .success(response):
+                self.initializeSession(with: response.sessionId, data: response.sessionData)
+            case let .failure(error):
+                self.presentAlert(with: error) { [weak self] in
+                    self?.setupSession()
+                }
+            }
+        }
+    }
+    
+    private func initializeSession(with sessionId: String, data: String) {
+        let configuration = AdyenSession.Configuration(sessionIdentifier: sessionId,
+                                                       initialSessionData: data,
+                                                       apiContext: ConfigurationConstants.apiContext)
+        AdyenSession.initialize(with: configuration, delegate: self, presentationDelegate: self) { [weak self] result in
```

Comment:
Would be nice to make `async` Demo integration 🤔

Reply:
For the refactor of the demo app, yes for sure!